src/views/EquipmentPage.vue

```diff
@@ -180,23 +181,25 @@ const handleDrop = async (slotType: string, index: number | null, event: DragEve
   const item = JSON.parse(itemData)
 
   if (item && item.slotType === slotType) {
```

Comment:
Few things:

- I don't really like else statements. Would prefer to remove those.
- You are now calling equipItem twice? Once on 'swap' and dagain after the last if statement.
- Any reason you decided to make dragOrigin an object, and not just a string?

Reply:
1. oh okay sorry i can do that
2. yes because the first one is moving the other accessory while the second one is the actual item equip
3. it was already sometimes a string (accessed through `dragOrigin.value`) and sometimes an object (accessed through `dragOrigin.value.slotType` -- namely [line 192-193 of the file before my commit](https://github.com/svecs132/idle-ant-farm/blob/ae2031c6a5327b9487da7676473cf4b7f60f90bc/src/views/EquipmentPage.vue#L192))

Reply:
ps. 3. it not only contains the slot type but also an index in the case of it being an accessory

Reply:
ps 1. how would you remove the else here without making it more complicated though? of course i could turn it into an if with `!==` instead of `===` but i dont think thats better

Reply:
I'll look at it later and fix it, thanks!